scala-package/core/src/main/scala/org/apache/mxnet/javaapi/NDArray.scala

```diff
@@ -108,28 +207,88 @@ class NDArray(val nd : org.apache.mxnet.NDArray ) {
 
   def serialize() : Array[Byte] = nd.serialize()
 
+  /**
+    * Release the native memory. <br />
+    * The NDArrays it depends on will NOT be disposed. <br />
+    * The object shall never be used after it is disposed.
+    */
   def dispose() : Unit = nd.dispose()
```

Comment:
I am contemplating if we ever need to expose this at all or should we direct users to use it along with ResourceScope to make the user experience better